### PR TITLE
improve coverage by adding specs and deleting vestigial and impossibl…

### PR DESCRIPTION
**Pull request by titusfortner**
I'm starting to go through all of the "uncovered" lines of code. Some of this code isn't needed, some of it needs specs added. I'm going to try not to wait until I've overhauled everything to get things merged in, but would appreciate another set of eyes on it.

### CODE REVIEW

File: lib/watir/cookies.rb [hunk 1]
```diff
@@ -119,15 +119,5 @@ def load(file = '.cookies')
         add(c.delete(:name), c.delete(:value), c)
       end
     end
-
-    private
-
-    def to_time(time)
-      if time.respond_to?(:to_time)
```
Comment (p0deje) [inline lib/watir/cookies.rb:126]: Do you know why this was needed?

File: lib/watir/locators/element/selector_builder.rb [hunk 2]
```diff
@@ -54,9 +54,9 @@ def build(selector)
           inspect = selector.inspect
           return given_xpath_or_css(selector) if selector.key?(:xpath) || selector.key?(:css)
 
-          built = build_wd_selector(selector)
-          Watir.logger.debug "Converted #{inspect} to #{built}"
-          built
+          build_wd_selector(selector).tap do |built|
```
Comment (p0deje) [inline lib/watir/locators/element/selector_builder.rb]: 🤷‍♂️

**Review by p0deje — APPROVED** [review]

File: lib/watir/cookies.rb (hunk 1 above)
Comment (titusfortner) [reply lib/watir/cookies.rb:126]: I spent a solid half hour trying to figure out what class instance would respond to year / month methods and not to_time. Then I realized that this is only acting on information coming back from selenium, so we know that what the object will be nil or date time

**Review by jkotests — APPROVED** [review]

File: lib/watir/cookies.rb (hunk 1 above)
Comment (p0deje) [reply lib/watir/cookies.rb:126]: It is possible this exists for backwards compatibility with older Ruby, but whatever.

Comment (titusfortner) [reply lib/watir/cookies.rb:126]: I literally couldn't find a class for any version of Ruby where it responded to `#year` but not `#to_time`. Maybe Selenium returned a custom something at one point?

File: lib/watir/locators/element/selector_builder.rb (hunk 2 above)
Comment (titusfortner) [reply lib/watir/locators/element/selector_builder.rb]: I know, this goes against my preference of not unnecessarily using blocks.

File: lib/watir/cookies.rb (hunk 1 above)
Comment (p0deje) [reply lib/watir/cookies.rb:126]: 🤷‍♂️

**Pull request merged**